docopt.py

```diff
@@ -13,6 +13,11 @@
 __all__ = ['docopt']
 __version__ = '0.6.1'
 
+PY3 = sys.version_info[0] == 3
+if PY3:
```

Comment:
I think duck-checking for `basestring` would be better than explicitly looking for Python 3 (some shim might reintroduce basestring into a py3 environment, or Python 4 might come out one day...):

```python
try:
    string_types = basestring,
except NameError:
    string_types = str,
```